app/services/doc_auth/lexis_nexis/error_generator.rb

```diff
@@ -56,48 +59,89 @@ class ErrorGenerator
       # rubocop:enable Layout/LineLength
 
       # rubocop:disable Metrics/PerceivedComplexity
-      def self.generate_trueid_errors(response_info, liveness_checking_enabled)
+      def self.generate_trueid_errors(response_info, liveness_enabled)
+        user_error_count = response_info[:AlertFailureCount]
+
+        errors = get_error_messages(liveness_enabled, response_info)
+        user_error_count += 1 if errors.include?(:SELFIE)
```

Comment:
On second thought, we should probably just remove the constants like `SELFIE` if they just represent simple symbols anyways?

Reply:
Yeah, good catch. 😵

Reply:
The original purpose of them was to allow it to be easier to change the hash symbol name for the front end while everything was still in flux. I have since grown to like it because I have found it much more readable in the code to pick out the all caps constants as I am  scanning the code.